person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private :is_of_age
```

Comment:
- [ ] Kindly change the name of the method to be `of_age?` instead of `is_of_age`  as there is no method with this name.

Reply:
@gemmen29 it's obliged by the linters.

Reply:
@teckim Yes, this is only typo error change the name of the function the same as after private but now the function name is `of_age?` and after private `:is_of_age`. make both with the same name.